uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-widget/leave-widget.component.js

```diff
@@ -1,19 +1,108 @@
 /* eslint-env amd */
 
 define([
-  'leave-absences/shared/modules/components'
-], function (components) {
+  'common/lodash',
+  'leave-absences/shared/modules/components',
+  'leave-absences/shared/models/absence-period.model',
+  'leave-absences/shared/models/absence-type.model',
+  './leave-widget-balance.component'
```

Comment:
Shouldn't the path be specified in the conf file? Just wondering.

Reply:
Paths can be relative.